src/routes/store.jsx

```diff
@@ -95,10 +82,10 @@ export default function Store() {
               width="200"
               height="200"
             /> : <div style={{ fontSize: 30, overflowWrap: "break-word" }} >{clearance.product.description}</div>}
-            <span className="bottom-left"><p>{clearance.offer.stock} tilbage</p><p>{dateText(clearance.offer.lastUpdate)}</p></span>
+            <span className="bottom left"><p>{clearance.offer.stock} tilbage</p><p>{dateText(clearance.offer.lastUpdate)}</p></span>
             {/*<span className="percentage">% {clearance.offer.percentDiscount}</span>*/}
             {/*<span className="original-price">før {clearance.offer.originalPrice},-</span>*/}
-            <span className="bottom-right"><p>Før <b>{clearance.offer.originalPrice}</b>,-</p><p> Nu kun <b>{clearance.offer.newPrice}</b>,-</p></span>
+            <span className="bottom right"><p>Før <b>{clearance.offer.originalPrice}</b>,-</p><p> Nu kun <b>{clearance.offer.newPrice}</b>,-</p></span>
```

Comment:
Har kikket på diff på denne linje i en time nu og kan ikke se nogen forskel. Is this some kind of joke?